guides/contexts.md

```diff
@@ -493,30 +493,27 @@ Next, let's expose our new feature to the web by adding the category input to ou
 defmodule HelloWeb.ProductHTML do
   use HelloWeb, :html
 
-  def category_select(f, changeset) do
+  def category_select(changeset) do
```

Comment:
Just nitpicking but it could be renamed to `category_options` with those changes:

```suggestion
  def category_options(changeset) do
```

Reply:
Makes sense to me.